mvg/analysis_classes.py

```diff
@@ -188,18 +254,32 @@ def summary(self):
         self.check_status()
         # print time info if applicable
         if self.time_column is not None:
-            from_t = self.dframe[self.time_column].min()
-            to_t = self.dframe[self.time_column].max()
+            from_t = self._dframe[self.time_column].min()
+            to_t = self._dframe[self.time_column].max()
             if self.time_column == "datetime":
                 from_t = from_t.strftime("%Y%m%d-%H:%M.%S")
                 to_t = to_t.strftime("%Y%m%d-%H:%M.%S")
             print(f"from {from_t} to {to_t}")
 
     # Default method
-    def plot(self):
-        """ Pro forma ancestor function"""
+    def plot(self, interactive=True):  # pylint: disable=unused-argument
+        """Pro forma ancestor function.
+
+        Parameters
+        ----------
+        interactive : bool
+            True: show plot, False: save plot
+
+
+        Returns
+        -------
+        plot file name: str
+          name of plot file (or emtpy string in case of interactive plot)
+        """
+
         self.check_status()
         print(f"Plot function not implemented for {type(self).__name__}")
+        return ""
 
     # Save self as pickel
     def save(self, file_name=None):
```

Comment:
Should probably be named `save_pkl`. Also, clarify in the parameters section of the docstring that `file_name` will default to `'<request_id>.pkl'`

Reply:
I'll fix it the documentation is already OK